lockbox-ios/Action/RouteAction.swift

```diff
@@ -29,6 +29,34 @@ enum MainRouteAction: RouteAction {
     case detail(itemId: String)
 }
 
+extension MainRouteAction: TelemetryAction {
+    var eventMethod: TelemetryEventMethod {
+        return .show
+    }
+
+    var eventObject: TelemetryEventObject {
+        switch self {
+        case .list:
+            return .entryList
+        case .detail:
+            return .entryDetail
+        }
+    }
+
+    var value: String? {
+        return nil
+    }
+
+    var extras: [String: Any?]? {
```

Comment:
These extras are not covered by any tests.

Reply:
@sasha how do i do something like: 

```swift
it("returns item id as extra") {
     expect(MainRouteAction.detail(itemId: "sfddfs").extras).to(equal([ExtraKey.itemid.rawValue: "sfddfs"]))
}

Reply:
@irrationalagent maybe try .to(contain(x)) ?

Reply:
😞 feel defeated by this... I've tried a many things but can't get something to work... especially these seem like they should work but im obviously in the dark when it comes to typing in swift

```swift
expect(MainRouteAction.detail(itemId: "sfddfs").extras).to(equal(["itemid": "sfddfs"]))
```
`Cannot convert value of type 'Predicate<[String : String]>' to expected argument type 'Predicate<[String : Any?]>'`

```swift
expect(MainRouteAction.detail(itemId: "sfddfs").extras!.Values).to(contain("sfddfs"))
```
or
```swift
expect(MainRouteAction.detail(itemId: "sfddfs").extras!.Values).to(equal(["sfddfs"]))
```
`Static member 'Values' cannot be used on instance of type '[String : Any?]'`

I considered converting the schema(?) for extras to be 

```swift 
var extras: [String: String?]? { get }
```

but I'd prefer not to since some extras might be integers in the future and it would involve changing a bunch of other lines as well... plz halp